rules/windows/c2_grey_list_beacon_sequence.toml

```diff
@@ -0,0 +1,88 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network activity from unexpected system applications. This may indicate adversarial activity as these
+applications are often leveraged by adversaries to execute code and evade detection.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Unusual Process Network Connection: Sequence"
```

Comment:
This name should be more informative.
Maybe "Unusual Network Activity from LOLBIN" or something to that effect

Reply:
```suggestion
name = "Unusual Network Activity from a LOLBIN"
```